content/kotsadm/installing/installing-embedded-cluster.md

```diff
@@ -6,11 +6,11 @@ weight: 10040
 draft: false
 ---
 
-This article refers to installing the Admin Console on an embedded cluster. 
+This article refers to installing the Admin Console on an embedded cluster.
```

Comment:
Not sure if this is better, but I feel like we should clarify that this creates a cluster with admin console, not installs admin console into a cluster.

```suggestion
This article refers to installing the Admin Console with an embedded cluster.
```

Reply:
Maybe "along with..."